CMakeLists.txt

```diff
@@ -43,6 +43,7 @@ option(UR_USE_TSAN "enable ThreadSanitizer" OFF)
 option(UR_ENABLE_TRACING "enable api tracing through xpti" OFF)
 option(UR_ENABLE_SANITIZER "enable device sanitizer" ON)
 option(UR_ENABLE_SYMBOLIZER "enable symoblizer for sanitizer" OFF)
+option(UR_ENABLE_EXCEPTION_SANITIZER "enable exception sanitizer" OFF)
```

Comment:
Not sure we need a separate option, we could always enable this as part of `UR_DEVELOPER_MODE` since that's enabled in CI.

Reply:
Thanks, have removed.